body.tex

```diff
@@ -225,16 +227,71 @@ \subsubsection{Team Leads and other Leadership Roles}
 The ratio of hours to \glspl{SP} remains constant, but the number of hours decreases.
 
 
-% should be explain the "annual planning" process here and emphasize what the Agencies (NSF/NOIRLab) are asking us to report?  Milestones, Milestone Activities, Deliverables.  I could imagine have a brief section describing annual planning, a section on progress tracking, a section on reporting.  We should also comment on the difference between the DOE and NSF side regarding FWP and annual planning timing.  It works in our favor, but we could summarize that in the way Phil did in his JODR presentation.
+% should we explain the "annual planning" process here and emphasize what the Agencies (NSF/NOIRLab) are asking us to report?  Milestones, Milestone Activities, Deliverables.  I could imagine have a brief section describing annual planning, a section on progress tracking, a section on reporting.  We should also comment on the difference between the DOE and NSF side regarding FWP and annual planning timing.  It works in our favor, but we could summarize that in the way Phil did in his JODR presentation.
 \section{Long Term Planning}
 \label{sec:long-term-plan}
 
+\subsection{Timeline for Annual Planning}
+\label{sec:annual-planning-timeline}
+
+% This might be a good place to discuss the linked timing of milestone development and budget scrubbing which I have a draft timeline here
```

Comment:
Good. The budget planning section I am working on also has a timeline in it, but it can usefully refer to this section once this is merged. I see some changes needed but it will be easier to make them once we merge this PR first.